### DISCUSSION TITLE
does the rocketmq-client-java client support broadcast mode consumption messages in rocketmq 5.0？

### DISCUSSION THREAD

**Discussion by chenyi19851209** [apache/rocketmq#6725] [Q&A]
Hello, in the proxy cluster mode of rocketmq 5.0, does the rocketmq-client-java client support broadcast mode consumption messages based on the grpc protocol?

**Comment by chenzlalvin** [comment]
Yes, we will provide atomic API interfaces to facilitate the implementation of one-to-N consumption scenarios, but there may not be broadcast mode parameters in version 4.x. cc to @aaron-ai
